app/helpers/patient_query_helper.rb

```diff
@@ -282,6 +282,18 @@ def advanced_filter(patients, filters, tz_offset)
         patients = advanced_filter_relative_date(patients, :symptom_onset, filter, tz_diff, :date)
       when 'continous-exposure'
         patients = patients.where(continuous_exposure: filter[:value].present? ? true : [nil, false])
+      when 'close-contact-with-known-case-id'
+        if filter[:value].blank?
+          patients = patients.where(contact_of_known_case_id: [nil, ''])
+        else
+          value_string = filter[:value].split(/\s*,\s*/).join('|')
+          case filter[:additionalFilterOption]
+          when 'Exact Match'
+            patients = patients.where('contact_of_known_case_id REGEXP ?', "(^|,\s?)(#{value_string})(\s?,|$)")
```

Comment:
Could you add a comment for the sake of future devs that explains this regex?

Reply:
And just generally comment on the support for multiple values here - might be confusing to see "Exact Match" and then a regex that is much more complicated than that haha

Reply:
done.  what else would you call it? it matches the value in the dropdown

Reply:
Oh I wasn't suggesting renaming, just explaining why a comment explaining why "Exact Match" has support for comma-separated values in the code would be good, which you just did, so all set